File: src_a_(C++)/Test4-8.c
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 4:ОПЕРАТОРЫ
//Тест номер    8:ДИРЕКТИВА FROM

include Test4_8a

char s[15];

void main() {
  lstrcpy(s,"frag1 ");
  lstrcat(s,"frag2");
  MessageBox(0,s,"fraf1 frag2",0);
}

File: src_a_(Pascal)/Test4-8.pas
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 4:ОПЕРАТОРЫ
//Тест номер    8:ДИРЕКТИВА FROM
program Test4_8;
uses Test4_8a;

var s:string[15];

begin
  lstrcpy(s,'frag1 ');
  lstrcat(s,'frag2');
  MessageBox(0,s,'fraf1 frag2',0);
end.

